### ISSUE TITLE
KL loss

### ISSUE THREAD

**Issue by maozhiqiang** [bfs18/nsynth_wavenet#6]
the H_Ps should be used  
![screenshot from 2018-07-06 11-31-11](https://user-images.githubusercontent.com/23501322/42358634-2271e57e-8110-11e8-8410-fd5e2d56381d.png)
so 
H_Ps = tf.reduce_mean(log_scale) + 2
change to H_Ps = tf.reduce_mean(log_scale) + 2*T
Is that right to understand

**Comment by HallidayReadyOne** [comment]
@maozhiqiang    @bfs18 
I am also a bit confused here, why not *T?
